File: src/main/java/valyush/platform/repository/LessonRepository.java
package valyush.platform.repository;

import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;
import valyush.platform.entity.Lesson;

import java.util.List;

@Repository
public interface LessonRepository extends JpaRepository<Lesson, Long> {
    List<Lesson> findByCourseModuleId(Long courseModuleId);
}

File: src/main/kotlin/v/orm/domain/lesson/repository/LessonRepository.kt
package v.orm.domain.lesson.repository

import org.springframework.data.jpa.repository.JpaRepository
import v.orm.domain.lesson.entity.Lesson
import java.util.*

interface LessonRepository : JpaRepository<Lesson, UUID> {
    fun findByModule_Id(moduleId: UUID): List<Lesson>
}